### PR TITLE
NTR vaping behavior

### PR DESCRIPTION
**Pull request by DitchingIt**
Creates a new term and child relations to tobacco and cannabis vaping as requested in issue #195

### CODE REVIEW

**Comment by pmidford** [comment]
Are we still accepting behaviors that are more or less exclusive to humans and occasional model organisms?  I thought there was an alternative ontology for these.

**Comment by DitchingIt** [comment]
> Are we still accepting behaviors that are more or less exclusive to humans and occasional model organisms? I thought there was an alternative ontology for these.

Hi @pmidford There aren't yet any plans to deprecate human or animal model behaviours, although it has been discussed. I'm not putting my own efforts into these areas but am happy to encourage others wanting to interact with NBO by helping with the odd thing like a commit, not least where all the thinking has already been done (like here), but you'll note I'm not attempting to impose the styling I'd look for if it was an Ethology edit. Would you be happy to approve this? If OP had made the commit, I would have, but because I wrote it up, GitHub needs someone else.

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -1053,6 +1053,7 @@ Declaration(Class(obo:UBERON_0002101))
 Declaration(Class(obo:UBERON_0002104))
 Declaration(Class(obo:UBERON_0002105))
 Declaration(Class(obo:UBERON_0005725))
+Declaration(Class(<http://purl.obolibrary.org/obo/nbo.owl/NBO_0070027>))
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:1056]: Something went wront with your protege config.

1. Open nbo on this branch
1. Click on the new class
2. Click (in menu) on "refactor"-->"rename entity"
3. Click "show full IRI"
4. Change to `http://purl.obolibrary.org/obo/NBO_0070027` (without the nbo.owl)
5. Safe, commit, push

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by matthewhorridge — APPROVED** [review]
This looks good.  @DitchingIt thanks so much for this and for the really quick turnaround time.

**Comment by DitchingIt** [comment]
@pmidford @matthewhorridge Can someone review and approve the changes I made resolving @matentzn change request? Then I think we're done.

**Review by matentzn — APPROVED** [review]
looks great

**Pull request merged**